src/Blaster.WebApi/Startup.cs

```diff
@@ -97,14 +98,20 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseForwardedHeadersAsBasePath();
             app.UseMetricServer();
-            app.UseStaticFiles();
             app.UseCorrelationId(new CorrelationIdOptions
             {
                 Header = "x-correlation-id",
                 UpdateTraceIdentifier = true,
                 IncludeInResponse = true,
                 UseGuidForCorrelationId = true
             });
+            
+            FileExtensionContentTypeProvider fileExtensionContentTypeProvider = new FileExtensionContentTypeProvider();
+            fileExtensionContentTypeProvider.Mappings[".webmanifest"] = "application/manifest+json";

```

Comment:
is the +json part required. I don't see it anywhere in the aspnet implementation: 
https://github.com/aspnet/AspNetCore/blob/def25cb3de048bd1127ab8a60ee84c0f6ba0da28/src/Middleware/StaticFiles/src/FileExtensionContentTypeProvider.cs